### DISCUSSION TITLE
Rounded corners on W11

### DISCUSSION THREAD

**Discussion by McNinjaTNT** [Simnico99/MicaWPF#41] [Q&A] [answered]
After implementing MicaWindow the corners become rounded on W11. I'm wondering if there is a way to control this, or better yet control the corner radius of the window.

**Comment by Simnico99** [comment, marked as answer]
Well the API used for the mica effect on windows 11 rounds the corner by default since it's a part of the OS Itself it can't be modified.

**Reply by McNinjaTNT** [reply]
I see, thanks for the information.

**Reply by Simnico99** [reply]
No problem don't hesitate if you have further questions 😊
